qiita_pet/handlers/analysis_handlers.py

```diff
@@ -141,14 +142,17 @@ def get(self, analysis_id):
             jobres[jobject.datatype].append((jobject.command[0],
                                              jobject.results))
 
-        dropped = {}
         dropped_samples = analysis.dropped_samples
+        dropped = defaultdict(list)
         if dropped_samples:
```

Comment:
Agreed -- thanks for pointing that out, I didn't look carefully at `analysis.dropped_samples` before commenting.